analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -19,6 +19,7 @@
  */
```

Comment:
This list of test cases is not meant to be exhaustive. We do have other tests for that. This should demonstrate though what symbols are computed.